pyapns_client/notification.py

```diff
@@ -303,8 +311,8 @@ def to_dict(self, alert_body=None) -> Dict[str, Any]:
 
 
 class _Notification:
-    PRIORITY_HIGH = 10
-    PRIORITY_LOW = 5
+    PRIORITY_HIGH = "10"
```

Comment:
I don't think this change is necessary because, at its core, priority is a number, not a string. Additionally, you added a conversion to string in the `get_headers` method.

Reply:
Done